src/pytest_html/basereport.py

```diff
@@ -18,7 +18,9 @@
 
 class BaseReport:
     def __init__(self, report_path, config, report_data, template, css):
-        self._report_path = Path(os.path.expandvars(report_path)).expanduser()
+        self._report_path = (
+            Path(os.path.expandvars(report_path)).expanduser().absolute()
```

Comment:
Not sure which is more correct [absolute](https://docs.python.org/3/library/pathlib.html#pathlib.Path.absolute) or [resolve](https://docs.python.org/3/library/pathlib.html#pathlib.Path.resolve) ?

Reply:
https://github.com/pytest-dev/pytest-html/blob/master/src/pytest_html/basereport.py#L189

Regardless, `resolve()` can be removed here.

Reply:
https://stackoverflow.com/questions/42513056/how-to-get-absolute-path-of-a-pathlib-path-object

Reply:
`resolve()` should be used to ensure the path is canonical

Reply:
Changed to `resolve`